Tests/CatalogItems/Write-RsFolderContent.Tests.ps1

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+Describe "Write-RsFolderContent" {
+
+    Context "Write-RsFolderContent with min parameters"{
+        $folderName = 'SutWriteRsFolderContentMinParameters' + [guid]::NewGuid()
+        New-RsFolder -Path / -FolderName $folderName
+        $folderPath = '/' + $folderName
+        $localReportPath =   (Get-Item -Path ".\").FullName  + '\Tests\CatalogItems\testResources'
+        Write-RsFolderContent -Path $localReportPath -RsFolder $folderPath
+        
+        It "Should upload a local report in Report Server" {
+            $uploadedReport = (Get-RsFolderContent -RsFolder $folderPath -Recurse ) | Where-Object TypeName -eq 'Report'
+            $uploadedReport.TypeName | Should Be 'Report'
+        }
+
+        It "Should upload a local RsDataSoutce in Report Server" {
+            $uploadedDataSource = (Get-RsFolderContent -RsFolder $folderPath -Recurse ) | Where-Object TypeName -eq 'DataSource'
+            $uploadedDataSource.TypeName | Should Be 'DataSource'
+        }
+
+        It "Should upload a local DataSet in Report Server" {
+            $uploadedDataSet = (Get-RsFolderContent -RsFolder $folderPath -Recurse ) | Where-Object TypeName -eq 'DataSet'
+            $uploadedDataSet.TypeName | Should Be 'DataSet'
+        }
```

Comment:
You are also have a data source in the test resources , you might want to add the validation that it is also uploaded into the server